### ISSUE TITLE
A11y_SqlDatabaseProjectsExtension_DatabaseProject_Keyboard: Tooltips provided for "New, Open existing, refresh, view and view more" controls are not accessible using keyboard.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#23549]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #Benchmark; #Keyboard; #WCAG1.1.1; #FTP; #SH-SQL Azure Data Studio-Linux-Jul23; #SQL Azure Data Studio; #Linux; 

Environment Details:
Application Name: Sql Database Projects Extension
Operating System: Ubuntu 20.04.2 LTS

### Repro Steps:
1. Launch Azure Data Studio application.
2. TAB to 'Extension' and search for “SQL Database Projects” and install it.
3. TAB to "Database project" and press ENTER key.
4. TAB to "New, Open existing, refresh, view and view more" controls and observe the issue.

### Actual Result:
Tooltips provided for "New, Open existing, refresh, view and view more" controls are not accessible using keyboard.

### Expected Result:
Tooltips provided for "New, Open existing, refresh, view and view more" controls should be accessible using keyboard.

### User Impact:
Keyboard users will not get information about the controls as tooltips are not accessible using keyboard.

### Recommendations:
Refer [Repository of bug fixes code snippets](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faka.ms%2faccsnippet&data=05%7c01%7cv-samhithaks%40microsoft.com%7c535b4a834ef843acdfb508da53769f91%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637914066217264155%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=xoIfLB9Iq6HVIPyUc42rS%2BlVQCt9m%2BaUokp32Kaz1nE%3D&reserved=0).

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

**Comment by kisantia** [comment]
@PoojaNamde this is an external issue and we already have an issue tracking this in ADS: https://github.com/microsoft/azuredatastudio/issues/19566, please lower the severity to Sev3 as the main issue is a Sev 3.

**Comment by PoojaNamde** [comment]
As per the https://github.com/microsoft/azuredatastudio/issues/19566, lowering the severity of the bug to Sev3.

**Comment by kisantia** [comment]
@PoojaNamde this is still showing as a sev2 in our dashboard. Can you please update the severity of this bug to Sev3 in the issue description and anywhere else this is tracked?

**Comment by PoojaNamde** [comment]
Lowered the severity to sev3.
